packages/now-cli/src/util/dev/server.ts

```diff
@@ -1651,7 +1652,6 @@ export default class DevServer {
     const env: EnvConfig = {
       ...process.env,
       ...this.buildEnv,
-      PATH: `${yarnBinPath}${delimiter}${process.env.PATH}`,
```

Comment:
@TooTallNate Don't we bundle `yarn` with now-cli for this exact purpose?

Reply:
Good question, I'm also curious about this.